vortex-ffi/src/lib.rs

```diff
@@ -10,21 +10,19 @@ mod file;
 mod log;
 mod stream;
 
-use std::cell::LazyCell;
 use std::ffi::{CStr, c_char, c_int};
+use std::sync::LazyLock;
 
 use tokio::runtime::{Builder, Runtime};
 use vortex::error::VortexExpect;
 
-thread_local! {
-    static RUNTIME: LazyCell<Runtime> = LazyCell::new(|| {
-        // Using a new_multi_thread runtime since a current local runtime has a deadlock.
-        Builder::new_multi_thread()
-            .enable_all()
-            .build()
-            .vortex_expect("building runtime")
-    });
-}
+static RUNTIME: LazyLock<Runtime> = LazyLock::new(|| {
+    // Using a new_multi_thread runtime since a current local runtime has a deadlock.
+    Builder::new_multi_thread()
+        .enable_all()
+        .build()
+        .vortex_expect("building runtime")
+});
```

Comment:
if you want something that is closer to the current_thread_runtime you can create a multithread runtime but with just a single worker thread IMO.

Reply:
IIRC, yes.

Reply:
Would that not mean all CPU for the whole query ran on a single thread? Even if there are n ddb threads

Reply:
It would also be just on thread if you ran it with current_thread_runtime (and this setup)